frontend/src/ThemedApp.tsx

```diff
@@ -13,8 +14,29 @@ import {
   setCachedTheme,
   ThemeConfig,
 } from "src/theme"
+import { zIndices } from "src/theme/primitives"
+
 import AppWithScreencast from "./App"
 
+function DataFrameOverlay(): React.ReactElement {
+  // The glide-data-grid requires one root level portal element for rendering the cell overlays:
+  // https://github.com/glideapps/glide-data-grid/blob/main/packages/core/API.md#htmlcss-prerequisites
+  // This is added to the body in ThemedApp
+
+  return (
```

Comment:
Hmm, not sure why I haven't used `useTheme` here. It seems to work fine, but I believe there was a problem when I implemented this.

Reply:
I had this code first within the widget/element folder of DataFrame... but I moved it to `ThemedApp` since I didn't want to do a full component import in `ThemedApp`. We `DataFrameOverlay` to its own `tsx` file, but I'm not sure about the best location. It does not really fit into `lib` and is not really big enough to be its own component.

Reply:
Feel free to check. I wonder if it has to do with the sidebar where we offer a change of theme?

I think this is just weird cause it operates outside of our normal styled components. Any reason this does not invoke Emotion styled components?

Reply:
I don't think this concern would bar a release since it's work and it's overall reasonable code, but we can do a followup exercise to ensure it meets our style guidelines.

Reply:
I think I had it outside the theme provider at first which did not work together with styled-components or useTheme. But I don't see any problem with the current state, so I changed it to be a styled component.

Reply:
👍